### ISSUE TITLE
Auto focus problem

### ISSUE THREAD

**Issue by jeancrus** [mkhstar/suneditor-react#23]
Hello! Im using suneditor on a form, and its starting with a auto focus on its fields. 
My code: 
```
import React from 'react';
import SunEditor from 'suneditor-react';
import PropTypes from 'prop-types';
import { Container } from './styles';

function SunEditorComponent({
    initialValue,
    onChange,
    onClickEditor,
    heightEditor,
    mode,
    placeHolder,
    ...rest
}) {
    return (
        <Container className="sun-editor-editable">
            <SunEditor
                {...rest}
                setContents={initialValue}
                onChange={onChange}
                lang="pt_br"
                onClick={onClickEditor}
                showToolbar={false}
                // show={false}
                autoFocus="false"
                // onImageUpload={handleImageUpload}
                setOptions={{
                    mode: 'inline',
                    buttonList: [
                        [
                            'undo',
                            'redo',
                            'font',
                            'fontSize',
                            'formatBlock',
                            'paragraphStyle',
                            'bold',
                            'underline',
                            'italic',
                            'strike',
                            'subscript',
                            'superscript',
                            'fontColor',
                            'hiliteColor',
                            'textStyle',
                            'removeFormat',
                            'outdent',
                            'indent',
                            'align',
                            'horizontalRule',
                            'list',
                            'lineHeight',
                            'table',
                            'link',
                            'image',
                            'video',
                            'fullScreen',
                            'showBlocks',
                            'codeView',
                            'print',
                            'save',
                        ],
                    ],
                }}
            />
        </Container>
    );
}

SunEditorComponent.propTypes = {
    initialValue: PropTypes.string.isRequired,
    mode: PropTypes.string,
    onChange: PropTypes.func,
    heightEditor: PropTypes.number,
    placeHolder: PropTypes.string,
};

SunEditorComponent.defaultProps = {
    heightEditor: null,
    onChange: null,
    mode: '',
    placeHolder: '',
};

export default SunEditorComponent;
```

And the one being called in the form 

```
<SunEditorComponent
                        onChange={value =>
                            handleChange(value, 'courseDescription')
                        }
                        initialValue={courseDescription}
                        placeHolder="Edite aqui sua página de descrição no curso"
                        heightEditor={230}
                    />
```
Have a workaround to remove the autofocus from it?
![image](https://user-images.githubusercontent.com/42682200/74460019-058fd180-4e6b-11ea-9431-96160037fbc7.png)

**Comment by mkhstar** [comment]
Fixed in the latest version suneditor-react@2.6.4
Use autoFocus={false} to disable.

**Comment by SwikarBhattarai** [comment]
If I set contents on initial state then there is still autofocus on every editor used. Please fix these too @mkhstar
